src/components/EncounterBuilderButton/EncounterBuilderButton.jsx

```diff
@@ -1,6 +1,18 @@
 import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
 import './EncounterBuilderButton.css';
-import { Link } from 'react-router-dom';
```

Comment:
Thanks for catching this, my mistake!

Reply:
Eh, I wouldn't even call it a mistake. We just probably didn't really need to add all those imports in until we're using them, since ya never know if it's gonna change when you start to implement. I'm sure there will be others that we'll just end up removing because we don't even use them. Nbd either way! 🙂